store/storetest/thread_store.go

```diff
@@ -679,7 +681,35 @@ func testGetTeamsUnreadForUser(t *testing.T, ss store.Store) {
 	assert.Equal(t, int64(1), teamsUnread[team2.Id].ThreadMentionCount)
 }
 
-func testGetThreadsForUser(t *testing.T, ss store.Store) {
+func testVarious(t *testing.T, ss store.Store) {
```

Comment:
I wanted to reuse the same "data structure" for these tests, while breaking them up, and found a helper function would make me return half a dozen created objects. Open to feedback for how to organize this better!